src/pages/TokenDetails/index.tsx

```diff
@@ -149,8 +149,9 @@ export default function TokenDetails() {
       })
     : null
 
-  const tokenData = query.tokenProjects?.[0]
-  const tokenDetails = tokenData?.markets?.[0]
+  const tokenProject = query.tokenProjects?.[0]
+  const tokenProjectMarket = tokenProject?.markets?.[0]
+  const tokenMarket = tokenProject?.tokens[0].market
```

Comment:
more descriptive names that match the fields being queried

Reply:
can we add a comment to devs above tokenProjectMarket and tokenMarket to describe the difference? Still semi unclear to me😅